### ISSUE TITLE
Wireless icon sometimes disappear

### ISSUE THREAD

**Issue by Quent42340** [jaagr/lemonbuddy#93]
Don't know if there's a problem in my config, but sometimes my wireless icon disappears.

Here is the related parts of my configuration file:

``` ini
[bar/top]
; ...
font-0 = Sans-Regular:size=8;0
font-1 = MaterialIcons:size=10;0
font-2 = FontAwesome:size=10;0
; ...

[module/wireless-network]
type = internal/network
interface = wlo1
interval = 3.0
ping-interval = 10

format-connected = <ramp-signal> <label-connected>
label-connected = %essid%
label-disconnected =    not connected
label-disconnected-foreground = #66

ramp-signal-0 = 
ramp-signal-1 = 
ramp-signal-2 = 
ramp-signal-3 = 
ramp-signal-4 = 
ramp-signal-5 = 

animation-packetloss-0 = 
animation-packetloss-0-foreground = #ffa64c
animation-packetloss-1 = 
animation-packetloss-1-foreground = ${bar/top.foreground}
animation-packetloss-framerate_ms = 500
```

For example when I'm very close to my router, I can't see the icon, but when I get away from it a little, the icon reappears. I was missing `ramp-signal-5` but even when I added it, the issue was still here.

**Comment by jaagr** [comment]
Try removing `ping-interval` to see if it's related to the packetloss state.

The default value for `format-packetloss = <label-connected>` which would only show the %essid% in your case if it entered that state... See the network module's wiki page for more details.

**Comment by Quent42340** [comment]
I tried removing `ping-interval`, but nothing has changed. :/

**Comment by jaagr** [comment]
Are you sure that all the icons exist in the given fonts? Try putting all of them in label-connected to see if they all appear.

**Comment by Quent42340** [comment]
Actually, only the `animation-packetloss-0` icon was different and I wondered if this wasn't the problem, but after setting the same exact icon everywhere, it still disappear...

**Comment by jaagr** [comment]
Could you post the full config you are using?

**Comment by Quent42340** [comment]
Yes, I'll give you the same link I gave on other tickets: https://github.com/Quent42340/dotfiles/blob/master/link/.config/lemonbuddy/config

**Comment by beanaroo** [comment]
I am experiencing the same thing.

```dosini
[module/wireless-network]
type = internal/network
interface = wlp3s0b1

label-connected = %local_ip%
label-disconnected = %Not connected%
label-disconnected-foreground = #66

format-connected = <ramp-signal> <label-connected>
ramp-signal-0 = 
ramp-signal-1 = 
ramp-signal-2 = 
ramp-signal-3 = 
ramp-signal-4 = 
ramp-signal-5 = 
```

**Comment by jaagr** [comment]
I'm reimplementing the event loop, which might fix this as well.. will put on hold until done with that

**Comment by jaagr** [comment]
Is this still valid using an up-to-date version?

**Comment by jaagr** [comment]
I cannot reproduce this, closing...

**Comment by rusmelezi** [comment]
I am experiencing same problem on latest version on polybar.

**Comment by patrick96** [comment]
This issue is extremely old. Please open a new one and provide both your config as well as the whole output of the polybar command you ran.
